docs/spec.yml

```diff
@@ -304,6 +303,260 @@ paths:
           description: Unprocessable request
         '500':
           description: Validation Error
+  '/v3/technical-records/updateVrm/{systemNumber}/{createdTimestamp}':
+    patch:
+      summary: 'Update VRM on a technical record'
+      tags:
+        - Update VRM
+      parameters:
+        - in: path
+          name: systemNumber
+          schema:
+            type: string
+            minLength: 3
+            maxLength: 21
+          required: true
+          description: This represents the systemNumber of the vehicle. Partition key of the DynamoDB table.
+        - in: path
+          name: createdTimestamp
+          schema:
+            type: string
+            pattern: \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{0,20}Z
+          required: true
+          description: Date at which the record was created. Sort key of the DynamoDB table.
+      operationId: updateVrm
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object 
+              required:
+                - newVrm
+              properties:
+                newVrm:
+                  type: string
+                isCherishedTransfer:
+                  type: boolean
+                thirdMark:
+                  type: string
+      responses:
+        '200':
+          description: Returns the updated technical record
+          content:
+            application/json:
+              schema:
+                $ref: "https://raw.githubusercontent.com/dvsa/cvs-type-definitions/v6.4.0/json-schemas/v3/tech-record/get/psv/complete/index.json"
+        '401':
+          description: Not authenticated
+        '403':
+          description: Unauthorised Access
+        '404':
+          description: Operation not supported
```

Comment:
A lot of the 404 responses have no resources match the search criteria. Do all of the 404's here, reflect that?